.github/release.yml

```diff
@@ -0,0 +1,17 @@
+changelog:
+  categories:
+    - title: Breaking Changes 🛠
+      labels:
+        - breaking-change
+    - title: New Features 🎉
+      labels:
+        - enhancement
+    - title: Bug Fixes
+      labels:
+        - bug
+    - title: Other Changes
+      labels:
+        - "*"
+    - title: ert3
+      labels:
+        - ert3
```

Comment:
Does this work like an if statement, or are items added to all matching cases?

Reply:
I need to move `ert3` up a level, as the `*` is:

> Use * as a catch-all for pull requests that didn't match any of the previous categories.

From the docs it looks like an if statement.